cmd/connect.go

```diff
@@ -0,0 +1,114 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+)
+
+func (h *Handler) Connect(ctx context.Context, req *entity.CommandRequest) error {
+	projectCfg, _ := h.cfg.GetProjectConfigs()
+
+	project, err := h.ctrl.GetProject(ctx, projectCfg.Project)
+	if err != nil {
+		return err
+	}
+
+	envName := getEnvironmentNameFromID(projectCfg.Environment, project.Environments)
+
+	fmt.Printf("🎉 Connecting to: %s %s\n", ui.MagentaText(project.Name), ui.MagentaText(envName))
+
+	names := make([]string, 0)
+	for _, plugin := range project.Plugins {
+		// TODO: Better way of handling this
+		if plugin.Name != "env" {
+			names = append(names, plugin.Name)
+		}
+	}
```

Comment:
This is a bit of an ugly thing here, but I just didn't see a better way to handle this...

Reply:
@coffee-cup Another env oopsie :<

Wonder if there's a way to move this before it becomes a "founding principle"

Reply:
e.g we ship binaries with logic for it

Reply:
We should probably have a `visiblePlugins` function that filters out env plugins. I can see at one point we would have multiple env plugins.

Reply:
could also just stop sending back "env" as part of the project plugins. But I imagine that would break a few other things

Reply:
Actually, in this case we might want a `getDatabasePlugins` since we will (hopefully) have other non-db plugins in the future